File: engine/model/src/main/scala/net/server/channel/packet/cash/operation/IncreaseStorageSlotsSmall.scala
package net.server.channel.packet.cash.operation

class IncreaseStorageSlotsSmall(private var _action: Int, private var _cash: Int, private var _mode: Byte) extends BaseCashOperationPacket(_action = _action) {
  def cash: Int = _cash

  def mode: Byte = _mode
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/IncreaseStorageSlotsSmall.java
package net.server.channel.packet.cash.operation;

public class IncreaseStorageSlotsSmall extends BaseCashOperationPacket {
   private final Integer cash;

   private final Byte mode;

   public IncreaseStorageSlotsSmall(Integer action, Integer cash, Byte mode) {
      super(action);
      this.cash = cash;
      this.mode = mode;
   }

   public Integer cash() {
      return cash;
   }

   public Byte mode() {
      return mode;
   }
}
